src/components/CallView/Grid/Grid.vue

```diff
@@ -475,7 +476,7 @@ export default {
 
 	watch: {
 		// If the video array size changes, rebuild the grid
-		'videos.length': function() {
+		'videos.length'() {
```

Comment:
Does this even works?
Can you watch length of arrays in vue watchers? 😮

Reply:
Seem to work fine, the grid rebuilds when someone joins :)